tests/trainer/test_trainer.py

```diff
@@ -3451,3 +3456,103 @@ def test_hyperparameter_search_backends(self):
             list(ALL_HYPERPARAMETER_SEARCH_BACKENDS.keys()),
             list(HPSearchBackend),
         )
+
+
+@require_peft
+@require_torch
+class TrainerPeftTest(unittest.TestCase, TrainerIntegrationCommon):
+    def get_regression_adapter_trainer(
+        self,
+        keep_report_to=False,
+        adapter_name="default",
+        adapter_args=None,
```

Comment:
by convention, args are passed as tuples, kwargs as dicts

```suggestion
        adapter_kwargs=None,
```